portfolio/src/main/webapp/feedback.html

```diff
@@ -23,8 +23,10 @@ <h1> Leave a Comment!</h1>
       <input type="text" id="name" name="title"></input><br>
       <label for="comment"> Comment: </label><br>
       <input type="text" id="comment" name="comment"> </input><br>
-      <input type="submit" value="Submit"> </input>
+      <input type="submit" value="Submit"> </input> <br><br>
       </form>
+             <label for="limit"> Number of Comments to Display: (Between 0 and 10) </label>
+             <input type="number" id="limit" name="limit" min="0" max="10" onchange="limitNumComments()">
```

Comment:
we tell our users that there is a max value of 10, but what if someone manages to pass in a number greater than 10? what if they passed in 999999999999? This might cause huge issues with your database and your server in returning this much information. We should do some server side validation to ensure that the max number is maintained.

Reply:
I added a check so if the number is greater than 10, it will still return 10.